packages/odie-client/src/utils/zendesk-message-converter.ts

```diff
@@ -59,7 +59,7 @@ export const zendeskMessageConverter: ( message: ZendeskMessage ) => Message = (
 	let type = message.type as MessageType;
```

Comment:
If the line 62 needs the `?` then this line would need it too.

Or

Maybe the `?`  is not needewd in `message?`

Also as I stated in Slack, we should update the ZendDeskMessage type to allow nullable in the source property :)